pipes/Broad_UGER/jobscript.sh

```diff
@@ -2,11 +2,14 @@
 # properties = {properties}
 # this is identical to the default jobscript with the exception of the exit code
 
+source /broad/software/scripts/useuse
+use Python-3.4
```

Comment:
Can this be omitted if the dotkit setup script is sourced above the pyyaml-dependent python calls?